Apps/phcanary/canary.json

```diff
@@ -0,0 +1,738 @@
+{
+    "appid": "00b3dc3a-ea6e-47a8-82a5-036f05d83863",
+    "name": "Canary",
+    "description": "App to Integrate with ThinkST Canary API Service",
+    "type": "siem",
+    "product_vendor": "ThinkST",
+    "logo": "logo_canary.svg",
+    "logo_dark": "logo_canary_dark.svg",
+    "product_name": "Canary",
+    "product_version_regex": ".*",
+    "publisher": "Splunk",
+    "license": "Copyright (c) 2019 Splunk Inc.",
+    "app_version": "1.0.2",
+    "utctime_updated": "2018-11-09T01:55:04.189254Z",
+    "package_name": "phantom_canary",
+    "main_module": "canary_connector.pyc",
+    "min_phantom_version": "4.5.15922",
+    "app_wizard_version": "1.0.0",
+    "configuration": {
+        "domain": {
+            "description": "Domain used for Canary Website. For example (https://*domain*.canary.tools)",
+            "data_type": "string",
+            "required": true,
+            "order": 0
+        },
+        "apiKey": {
```

Comment:
`api_key` please also change in connector file

Reply:
Fixed.